bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        use_stdout=True,
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts',
+                                            dnf='licensecheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=None,
+                         licensecheck_tail: int=None):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.(Default: 60)
```

Comment:
Another workaround that I think will help is :  
* If the user mentions `--lines 30`, we can set `--tail` to 0 to make sure that the file is parsed only for the first 30 lines.
* If the user mentions `--tail 200`, we can set `--lines` to 0 to make sure that the file is parsed only for the last 200 bytes of the file.
* If he mentions both, the results will be due to parsing from either sides, which we get in normal circumstances. 
Maybe that is what is happening in the upgraded version.
* So, if I specify `licensecheck --lines 20 xyz.py`, I might expect an error if the license is not defined within the first 20 lines but the default for `--tail` is 5000, due to which the file will also be parsed from the bottom. We can achieve that by setting the other option to 0. Maybe this is what they mean when they say `(and ignore --tail)` [here](paste.ubuntu.com/24102936/). Also, note jayvd's output [here](https://gitter.im/coala/coala-bears?at=58b99a7bf1a33b627576cabf).